cvpy/annotation/cvat/CVATProject.py

```diff
@@ -210,8 +210,41 @@ def resume(project_name: str, cas_connection: CAS, caslib: str, relative_path: s
             SPecifies the caslib under which CAS tables were saved.
         relative_path:
```

Comment:
Is there any way to fetch the relative path knowing the caslib name? It would be nice to eliminate the need to supply this _along _with_ the caslib name itself. 

If not, not big deal!

Reply:
The relative path is not the same as the caslib path. It is the path (subdirectories) under the caslib path. So we cannot eliminate this parameter. 

e.g. A user has a caslib='test' with path='/home/bob/data', and the user has saved a project in /home/bob/data/projects/cvat/MyDemoProject.sashdat. Then user will specify caslib=test and relative_path='projects/cvat'.